docs/_sources/real_time_messaging.rst.txt

```diff
@@ -27,7 +27,7 @@ Connecting to the Real Time Messaging API
           print sc.rtm_read()
           time.sleep(1)
   else:
-      print "Connection Failed, invalid token?"
+      print "Connection Failed"
```

Comment:
uhh.. this directory is supposedly ignored, you're still committing files here? might want to `git rm -rf docs/_sources` to get the actual delete as part of the commit